recipe/meta.yaml

```
@@ -17,31 +17,45 @@ build:
 requirements:
   host:
     - pip
-    - setuptools
-    - wheel
     - python
+    - wheel
+    - setuptools >=61.0.0
   run:
     - python
     - numpy >=1.23.2
     - pandas >=2.0.0
     - attrs >=19.3.0
     - networkx >=2.4
     - multimethod >=1.4
+    - puremagic
+    # imagehash - https://github.com/dylan-profiler/visions/blob/v0.8.1/src/visions/utils/images/image_utils.py#L5
+    - imagehash
+
+# pytest-spark >=0.6.0 is not available
+{% set tests_to_skip = "test_import_spark_session" %}
+{% set tests_to_skip = tests_to_skip + " or test_create_spark_session" %}
+{% set tests_to_skip = tests_to_skip + " or test_spark_backend" %}
 
 test:
+  source_files:
+    - pytest.ini
+    - tests
   imports:
     - visions
   commands:
     - pip check
+    - pytest -v -k "not ({{ tests_to_skip }})"
```

```suggestion
    - pytest -v -m "not spark_test"
```
This should be enough, because of https://github.com/dylan-profiler/visions/blob/v0.8.1/pytest.ini#L6C5-L6C15.
Then you can remove the test to skip.